### ISSUE TITLE
gnome3.geary doesn't build

### ISSUE THREAD

**Issue by dotlambda** [NixOS/nixpkgs#118309]
https://hydra.nixos.org/build/140600464

cc @NixOS/gnome

**Comment by lukegb** [comment]
From e.g. https://hydra.nixos.org/build/140600464/nixlog/1

```
WKError: Is false, expected true
Composer.PageStateTest/edit_context_font: This call should not be reached
not ok 60 /js/Composer.PageStateTest/edit_context_font
```

**Comment by mweinelt** [comment]
Caused by #117991, reported upstream at https://gitlab.gnome.org/GNOME/geary/-/issues/1180.

Fixed in #118326
